src/condor_tests/test_fto_failure_propagation.py

```diff
@@ -0,0 +1,843 @@
+#!/usr/bin/env pytest
+
+import pytest
+from ornithology import *
+import classad2
+
+#--------------------------------------------------------------------------
+# Test Case Structure:
+# TestName : (
+#     Submit Description Dictionary,
+#     Ornithology ClusterState Wait Condition,
+#     (
+#          Expected Plugin Return Ad Details,
+#          Plugin Transfer Ad Type (input/output),
+#          Expected Hold Code,
+#          Expected Hold SubCode,
```

Comment:
This is actually the expected exit code; the conversion to/from the hold subcode is handled automatically.  That's find (although a little confusing initially), but potentially limiting depending on how we decide to fix the handling of plug-ins killed by signals, and/or if we ever use the low 8 bits for things like "file not found."

Reply:
If we do use the lower eight bits in the future it should be trivial to modify the test appropriately